github/Requester.py

```diff
@@ -832,6 +832,51 @@ def __structuredFromJson(self, data: str) -> Any:
                     raise
                 return {"data": data}
 
+    def getFile(
+        self,
+        url: str,
+        path: str,
+        parameters: Optional[Dict[str, Any]] = None,
+        headers: Optional[Dict[str, str]] = None,
+        cnx: Optional[Union[HTTPRequestsConnectionClass, HTTPSRequestsConnectionClass]] = None,
+    ) -> None:
+        """
+        GET a file from the server and save it to the given path.
```

Comment:
Sounds like path is a directory, but path includes the file name.

Reply:
This is the definition of path (that I am using): https://en.wikipedia.org/wiki/Path_(computing)
path can mean the full path to a file (which is the usage I have in mind)
You are correct that path can also mean just the base path of a file, i.e., the parent directory.  Are you suggesting a different word for the parameter?

Reply:
making this more clear in the doc string is sufficient

Reply:
okay